portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,43 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> commentList;
+  private String json;
+
+  @Override
+  public void init() {
+    commentList = new ArrayList<>();
+    commentList.add("Hello Alisa!");
+    commentList.add("Hola");
+    commentList.add("How are you?");
+    json = convertToJson(commentList);
```

Comment:
you might want to do the convert in the doGet method as the commentList might have content change later when you implement the support for adding message to the list and you want to convert the up to date list to return to frontend for every Get request.